packages/frontend/src/components/Header.vue

```diff
@@ -1,7 +1,13 @@
 <template>
   <div>
     <v-app-bar class="elevation-0">
-      <v-toolbar-title>{{ headerTitle }}</v-toolbar-title>
+      <v-toolbar-title v-bind="attrs" v-on="on">{{ headerTitle }}</v-toolbar-title>
+      <v-tooltip v-if="headerInfo" bottom>
+        <template v-slot:activator="{ on }">
+          <v-icon v-on="on">mdi-information-outline</v-icon>
+        </template>
+        <span>{{ headerInfo }}</span>
+      </v-tooltip>
```

Comment:
The runtime warning thrown on this annotation: 
```javascript
vue.esm.js:628 [Vue warn]: Property or method "on" is not defined on the instance but referenced during render.<br>
Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property. See: https://vuejs.org/v2/guide/reactivity.html#Declaring-Reactive-Properties.

found in

---> <Header> at src/components/Header.vue
       <VApp>
         <App> at src/youi/App.vue
           <Root>
```

Reply:
`on` is required for material design tooltip to function. This is not an issue but a result of how vue works, since `on` hover should not be exposed to the parent component  I dont think we need to fix this?